recipes/slda/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "slda" %}
+{% set version = "0.1.6" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/Savvysherpa/slda/archive/{{ version }}.tar.gz
+    sha256: aeabfe4edefc566ee97c107f8f9b2c6a478595e24fb35d44d9eeaaa876fb90ce
+
+build:
+    number: 0
+    skip: True  # [win or not py3k]
+
+requirements:
+    build:
+        - cython
+        - cythongsl
```

Comment:
Maybe that has merit in the general case. Happy to discuss that with you elsewhere. However I don't think it applies here.

`cythongsl` is a header only library and as such does not break when the ABI of `gsl` changes. However, `slda` that links to `gsl` will be broken if `gsl`'s ABI changes. I don't think we are over-specifying by providing this additional detail about ABI sensitivity.